packages/core/upload/admin/src/pages/App/tests/MediaLibrary.test.js

```diff
@@ -225,6 +225,11 @@ describe('Media library homepage', () => {
       });
 
       it('hides the select all button when the user is not allowed to update', () => {
+        useMediaLibraryPermissions.mockReturnValueOnce({
```

Comment:
@joshuaellis I had weird issues on ML tests, `canUpdate` mock returned undefined sometimes breaking the display of `FolderCardCheckbox` - I'd love to have a deeper look at it together if you can!

Reply:
pair programmed on it ✅